### PR TITLE
Write Access with id ?

### PR DESCRIPTION
**Pull request by spartanz51**

### CODE REVIEW

**Comment by phillmac** [comment]
The id is not the publuc key. The public key is required for access control :cat:

**Comment by tabcat** [comment]
weird, ive always added the id to the accessController.write array

**Comment by tabcat** [comment]
these changes seem correct

**Comment by phillmac** [comment]
It used to be that the id was the public key, but recently it's become possible to define the public key and id separately, so id !== public key

**Comment by phillmac** [comment]
See  https://github.com/orbitdb/orbit-db-identity-provider/blob/e473da659b1ef657a77bf2314854dfb3838149ff/src/identity.js#L5

**Comment by aphelionz** [comment]
Thanks for the PR @spartanz51 .

These changes are indeed correct. You can see the tests here: https://github.com/orbitdb/orbit-db/search?q=%22.identity.id%22&unscoped_q=%22.identity.id%22

There was another discussion about this recently as well, in this PR. You can follow more of the discussion there. https://github.com/orbitdb/orbit-db-access-controllers/pull/31

File: GUIDE.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ ipfs.on('ready', async () => {
 })
 ```
 
-To give write access to another peer, you'll need to get their public key with some means. They'll need to give you the output of their OrbitDB instance's key: `orbitdb.identity.publicKey`.
+To give write access to another peer, you'll need to get their public key with some means. They'll need to give you the output of their OrbitDB instance's key: `orbitdb.identity.id`.
```
Comment (tabcat) [inline GUIDE.md]: should probably have changed 'OrbitDB instance's key' to 'OrbitDB instance's id'

**Comment by aphelionz** [comment]
Fixed it in post ;)

**Pull request merged**